Typora.Typora.Beta version 0.11.18

Dragon1573 opened the pull request:
## Prerequisite Check

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

## Note

There's a **closed (but not merged)** pull requests #38872, but it's closed because of duplicates and author unattention.

I think there're still many users don't want to pay for Typora v1.x, so I create this pull request and adding the latest beta version (`0.11.18`) of Typora. It may help others.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46076)

Dragon1573 committed:
Typora.Typora.Beta version 0.11.18

OfficialEsco commented on manifests/t/Typora/Typora/Beta/0.11.18/Typora.Typora.Beta.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageVersion: 0.11.18
+PackageLocale: en-US
+Publisher: typora.io
+PackageName: Typora
```
```suggestion
PackageName: Typora Beta
```

OfficialEsco reviewed (changes requested):
Weird, not getting any ARP Metadata, but the PackageName needs to change so it dosn't conflict with the stable version
The other PR got closed because the OP didn't reply

Dragon1573 committed:
Update manifests/t/Typora/Typora/Beta/0.11.18/Typora.Typora.Beta.locale.en-US.yaml

- Change the "PackageName" so it doesn't conflict with the stable version
- Suggested by @OfficialEsco

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>
